java/vector/src/main/java/org/apache/arrow/vector/TimeStampSecVector.java

```diff
@@ -112,9 +114,7 @@ public LocalDateTime getObject(int index) {
     } else {
       final long secs = valueBuffer.getLong(index * TYPE_WIDTH);
       final long millis = java.util.concurrent.TimeUnit.SECONDS.toMillis(secs);
-      final org.joda.time.LocalDateTime localDateTime = new org.joda.time.LocalDateTime(millis,
-              org.joda.time.DateTimeZone.UTC);
-      return localDateTime;
+      return DateUtility.getLocalDateTimeFromEpochMilli(millis);
```

Comment:
This change here is a change in API. It will break all applications using this getObject API.